web/tls_config.go

```diff
@@ -366,11 +373,19 @@ func Serve(l net.Listener, server *http.Server, flags *FlagConfig, logger *slog.
 		return err
 	}
 
+	var limiter *rate.Limiter
+	// Setup Rate Limiter
+	if c.RateLimiterConfig.Rate != 0 && c.RateLimiterConfig.Burst != 0 {
+		limiter = rate.NewLimiter(rate.Limit(c.RateLimiterConfig.Rate), c.RateLimiterConfig.Burst)
```

Comment:
Do we want to use `rate.Every()` here to allow configuring the window (X/min, X/hour)?

Reply:
Good point, I'll take a look.